NimbleSurvey/Sources/CommonViews/OverlayView.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  LinearGradientView.swift
+//  NimbleSurvey
+//
+//  Created by Doan Thieu on 28/09/2022.
+//
+
+import SwiftUI
+
+struct OverlayView: View {
+
+    var body: some View {
+        LinearGradient(
+            gradient: Gradient(colors: [.black.opacity(0.01), .black.opacity(0.8)]),
+            startPoint: .top,
+            endPoint: .bottom
+        )
+    }
+}
+
+struct LinearGradientView_Previews: PreviewProvider {
```

Comment:
Please consider to follow our conventions

Reply:
Updated in [#a6e9d0f](https://github.com/Thieurom/ic-surveys-ios/pull/31/commits/a6e9d0feefbf7fa5aa9ddd166519970026f5215d)